modules/refined/src/test/scala/doobie/refined/RefinedSuite.scala

```diff
@@ -54,15 +52,15 @@ class RefinedSuite extends munit.FunSuite {
   }
 
   test("Query should return a refined type when conversion is possible") {
-    sql"select 123".query[PositiveInt].unique.transact(xa).void.unsafeRunSync()
+    sql"select 123".query[PositiveInt].unique.transact(xa).void.assert
   }
 
   test("Query should return an Option of a refined type when query returns null-value") {
-    sql"select NULL".query[Option[PositiveInt]].unique.transact(xa).void.unsafeRunSync()
+    sql"select NULL".query[Option[PositiveInt]].unique.transact(xa).void.assert
   }
 
   test("Query should return an Option of a refined type when query returns a value and converion is possible") {
-    sql"select NULL".query[Option[PositiveInt]].unique.transact(xa).void.unsafeRunSync()
+    sql"select NULL".query[Option[PositiveInt]].unique.transact(xa).void.assert
```

Comment:
This test case does not correspond to the test title, I guess it should look like this:
```suggestion
    sql"select 123".query[Option[PositiveInt]].unique.transact(xa)
      .assertEquals(Some(PositiveInt(123))
```
Otherwise there's no difference with the previous one.

Reply:
Also there's a typo in the title, must be "conver***s***ion", I guess.

Reply:
Resolved Both